samples/benchmark/gamelife/kotlin/src/main/kotlin/godot/samples/benchmark/gamelife/game.kt

```diff
@@ -0,0 +1,176 @@
+package godot.samples.benchmark.gamelife
+
+import godot.*
+import godot.core.*
+import org.godotengine.kotlin.annotation.RegisterClass
+import org.godotengine.kotlin.annotation.RegisterFunction
+import org.godotengine.kotlin.annotation.RegisterProperty
+import kotlin.math.max
+import kotlin.math.sqrt
+
+@RegisterClass("scripts/gamelife")
+class Game: MultiMeshInstance(){
+
+    private lateinit var map : Map
+    private var started: Long = 0
+
+    private var counter : Int = 0
+    private var frame : Int = 0
+    private val rng = RandomNumberGenerator()
+
+    @RegisterProperty(false, "50")
+    var size = 50
+    @RegisterProperty(false, "1000")
+    var limit = 1000
+
+    @RegisterFunction
+    override fun _ready(){
+        started = OS.getSystemTimeMsecs()
+        map = Map(size, size)
+        multimesh.instanceCount = (map.w * map.h).toLong()
+
+        val l = sqrt(multimesh.instanceCount.toDouble()).toLong()
+
+        var index = 0L
+
+        val offset = (size - 50) * 0.5
+        for (y in 0 until l){
+            for (x in 0 until l){
+                multimesh.setInstanceTransform(index, Transform(Basis(), Vector3(x - offset, rng.randi() % 10 * 0.1, y - offset)))
+                multimesh.setInstanceColor(index, Color(255, x, y))
+                index += 1
+            }
+        }
+
+    }
+
+    @RegisterFunction
+    override fun _process(delta: Double){
+        frame += 1
+        if (frame == limit){
+            val elapsed = OS.getSystemTimeMsecs() - started
+            GD.print("Result: $elapsed")
+            getTree().quit()
+            return
+        }
+
+        counter += 1
+        if (counter == 10){
+            val center = size/2
+            map.put(center, center, true)
+            map.put(center-1, center, true)
+            map.put(center+1, center, true)
+            map.put(center, center-1, true)
+            counter = 0
```

Comment:
Formatting. Press once `Ctrl + Alt + L` or `Cmd + Alt + L` to auto format the script :-)